textproc/aha/Portfile

```
@@ -0,0 +1,25 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        theZiz aha 0.5.1
```

Can we also add a `revision` line:

```suggestion
github.setup        theZiz aha 0.5.1
revision            0
```

---

Ah, I though that one was unnecessary for now (`odt2txt` did not have it). I’ll add it back

---

The purpose of `revision` is so that if a change needs to be made to the built results of the port (binaries, config, etc.), but the `version` hasn't changed, then the change is signaled by incrementing `revision`.